controllers/admin/auth.controller.js

```diff
@@ -0,0 +1,99 @@
+const Joi = require("joi");
+const User = require("../../models/User");
+const bcrypt = require("bcrypt");
+const firebase = require("../../helpers/auth");
+const {
+  successResponse,
+  errorResponse,
+} = require("../../helpers/responseHelper");
+const {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+  sendEmailVerification,
+} = require("firebase/auth");
+
+const register = async (req, res) => {
```

Comment:
please add `(req, res, next)` if a variable is unused `(req, res, _next)` prefix `_` for un unused vars.